src/sklearn_evaluation/model_heuristics/model_heuristics.py

```diff
@@ -0,0 +1,147 @@
+from sklearn_evaluation.report.serialize import EvaluatorHTMLSerializer
+from sklearn_evaluation.report.report import Report
+from jinja2 import Template
+from sklearn_evaluation.report.serialize import figure2html
+import abc
+
+
+class ModelHeuristics(abc.ABC):
```

Comment:
we have a `report` module, let's move this there so the imports are consistent

`from sklearn_evaluatio.report import ...`